src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.CoreCLR.sfxproj

```diff
@@ -27,6 +27,8 @@
     <PublishReadyToRun Condition="'$(TargetOS)' == 'netbsd' or '$(TargetOS)' == 'illumos' or '$(TargetOS)' == 'solaris' or '$(TargetOS)' == 'haiku'">false</PublishReadyToRun>
     <!-- Disable crossgen on FreeBSD when cross building from Linux. -->
     <PublishReadyToRun Condition="'$(TargetOS)'=='freebsd' and '$(CrossBuild)'=='true'">false</PublishReadyToRun>
+    <!-- Disable crossgen for StageTwoBuild. -->
```

Comment:
Comments that just duplicate the code in plain English are not very useful. This comment should explain why we are doing this.

Reply:
It is consistent with line 28. R2R is disabled for staged builds in other places: https://github.com/dotnet/runtime/blob/03324cdb4ad5f408129ef203bc179caf55dbb642/src/coreclr/tools/aot/ILCompiler/ILCompiler_inbuild.csproj#L18 sfxproj is only built in stage2. I'll just delete the comment.